pal/os/inc/windows/az_ulib_pal_os.h

```diff
@@ -13,12 +13,34 @@ extern "C"
 #endif
 
   /*
-   *  @struct az_ulib_pal_os_lock
-   *
-   *  @brief  pointer to a platform specific struct for a lock implementation
+   *  @brief  Pointer to a platform specific lock handle.
    */
   typedef SRWLOCK az_ulib_pal_os_lock;
 
+  /*
+   *  @brief  Pointer to a platform specific thread handle.
+   */
+#define az_ulib_pal_thread_handle HANDLE
+
+  /*
+   *  @brief  Platform specific thread arguments type.
+   */
+#define az_ulib_pal_thread_args LPVOID
+
+  /*
+   *  @brief  Platform specific thread return type.
+   */
+#define az_ulib_pal_thread_ret DWORD WINAPI
```

Comment:
why change the windows pal?

Reply:
I didn't understand your question. We need thread in the PAL for the telemetry sample, and we need it for Linux and Windows.